library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
+    {
+        return $this->rateLimit;
+    }
+
+    /**
+     * Populates the object with info. This can possibly called from the
```

Comment:
this comment is about `populate()`, which should only happen at `__construct` (or lazily, later), but should not be exposed to userland